lib/EventTarget.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+
+const Node = require("./Node");
+const {EventEmitter} = require("events");
+
+const emitterSymbol = Symbol.for("emitter");
+const listenersSymbol = Symbol.for("listeners");
+
+module.exports = class EventTarget extends Node {
```

Comment:
_browsery_ [`EventTarget`](https://nodejs.org/api/events.html#eventtarget-and-event-api) has been supported by Node since v14.5.0.
events don't bubble however but maybe just extend the node interface with some custom behavior?